src/main/java/org/opentripplanner/util/HttpUtils.java

```diff
@@ -85,4 +98,28 @@ public static InputStream openInputStream(URI uri, Map<String, String> headers)
       return downloadUrl.openStream();
     }
   }
+
+  private static HttpResponse getResponse(
```

Comment:
This has nothing to do with your PR but now that we have this method we can actually use it in SiriHttpUtils.